### PR TITLE
Audio-based TN for Swedish

### PR DESCRIPTION
**Pull request by jimregan**
Audio-based TN for Swedish, for Språkbanken Tal

Replaces #48 

# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [X] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [X] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [X] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_lm.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                | pynutil.add_weight(pynini.compose(fraction_graph, v_fraction_graph), sem_w)
+                | pynutil.add_weight(pynini.compose(money_graph, v_money_graph), sem_w)
+                | pynutil.add_weight(cardinal_or_date_final, sem_w)
+                | pynutil.add_weight(whitelist_graph, sem_w)
+                | pynutil.add_weight(v_word_graph, 1.1001)
+            ).optimize()
+
+            classify_and_verbalize = pynutil.insert("< ") + classify_and_verbalize + pynutil.insert(" >")
+            classify_and_verbalize |= pynutil.add_weight(word_graph, word_w)
+
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_lm.py:157]: Is there a difference between this file and "tokenize_and_classify_with_audio"? For Eng, this file uses "lm" argument that alters graphs, e.g., limits the number of options for "lm=True".

File: tests/nemo_text_processing/sv/test_normalization_with_audio.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```
Comment (ekmb) [inline tests/nemo_text_processing/sv/test_normalization_with_audio.py:1]: 2022->2023

File: nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_lm.py [hunk 3]
```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```
[the remark this answers is not in the document]
Comment (jimregan) [reply nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_lm.py:1]: This is not a 'new' file, it's merely an adaptation of the same file in the English module.

File: nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_lm.py (hunk 1 above)
Comment (jimregan) [reply nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_lm.py:157]: Not at the moment. I'll drop it until there's a need for it.

File: nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_with_audio.py [hunk 4]
```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```
[the remark this answers is not in the document]
Comment (jimregan) [reply nemo_text_processing/text_normalization/sv/taggers/tokenize_and_classify_with_audio.py:1]: Also not "new"

File: tests/nemo_text_processing/sv/test_normalization_with_audio.py (hunk 2 above)
Comment (jimregan) [reply tests/nemo_text_processing/sv/test_normalization_with_audio.py:1]: Also not a "new" file

**Pull request merged**